docker/create_cluster.py

```diff
@@ -70,22 +65,48 @@ def create_cluster(clean, network_name, number_of_nodes, public_key_path):
     # Loop through nodes, start them on the network that was just created
     print('Starting {} containers on network {} ...'.format(number_of_nodes, network_name))
     container_names = ['{}.{}'.format(network_name, i) for i in range(number_of_nodes)]
+
+    is_first = True
     for container_name in container_names:
 
         print('Starting container: {} on network: {}'.format(container_name, network_name))
 
+        # Delete / Create tmp mount dir
+        tmp_dir = '/tmp/{}'.format(container_name)
+
+        if os.path.isdir(tmp_dir):
+            shutil.rmtree(tmp_dir)
+        os.mkdir(tmp_dir)
+
         # Priviledged is required for some ansible playbooks
         # The volume binding is needed for systemd
         # https://hub.docker.com/r/centos/systemd/
-        container = docker_client.containers.run(
-            'sethrosetter/centos7-systemd-sshd',
-            detach=True,
-            name=container_name,
-            privileged=True,
-            volumes={
-                '/sys/fs/cgroup': {'bind': '/sys/fs/cgroup', 'mode': 'ro'}
-            }
-        )
+        if is_first:
+            # Hack, Currently the first container will always be a Couchbase Server. We want to bind
+            # it to localhost so that we can debug using the Admin UI
+            container = docker_client.containers.run(
+                'sethrosetter/centos7-systemd-sshd',
+                detach=True,
+                name=container_name,
+                privileged=True,
+                volumes={
+                    '/sys/fs/cgroup': {'bind': '/sys/fs/cgroup', 'mode': 'ro'},
+                    tmp_dir: {'bind': '/home'}
+                },
+                ports={'8091/tcp': 33333}
+            )
+            is_first = False
+        else:
+            container = docker_client.containers.run(
+                'sethrosetter/centos7-systemd-sshd',
```

Comment:
I would like to propose that this docker clean up not block this PR as I will be working on it right after in its own branch and there will be a lot of changes.